index.html

```diff
@@ -1,9 +1,114 @@
 <html>
   <head>
     <meta charset="utf-8">
+    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet" />
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.6.3/css/font-awesome.min.css" rel="stylesheet" />
+    <link rel="stylesheet" href="css/style.css">
     <title>Zadanie 9.7</title>
   </head>
+
   <body>
+
+  <!-- Game title -->
+
+  <div class="container" >
+    <h3 class="text-center">
+      Welcome to the game
+      <span class="label label-default">Rock</span>
+      <span class="label label-info">Paper</span>
+      <span class="label label-danger">Scissors</span>
+    </h3>
+  </div>
+
+
+  <!-- Final winner -->
+  <div id="js-finalWinner" class="container text-center final">
+    
+  </div>
+
+  <!-- 'New game' button -->
+
+  <div id="js-newGameElement" class="container">
+    <div class="row text-center">
+      <button  id="js-newGameButton" class="btn btn-success">
+        New game
+      </button>
+    </div>
+  </div>
+
+  <!-- Player pick buttons -->
+
+  <div id="js-playerPickElement" class="container player-pick">
+    <div class="row text-center">
+      <button id="js-playerPick_rock" class="btn">
+        <span class="badge">
+          <i class="fa fa-hand-rock-o" aria-hidden="true"></i>
+        </span>
+        Rock
+      </button>
+      <button id="js-playerPick_paper"  class="btn">
+        <span class="badge">
+          <i class="fa fa-hand-paper-o" aria-hidden="true"></i>
+        </span>
+        Paper
+      </button>
+      <button id="js-playerPick_scissors"  class="btn">
+        <span class="badge">
+          <i class="fa fa-hand-scissors-o" aria-hidden="true"></i>
+        </span>
+        Scissors
+      </button>
+    </div>
+  </div>
+
+  <!-- Results table -->
+
+  <div id="js-resultsTableElement" class="container">
+
+    <div class="row"> <!-- row 1 -->
+      <div class="col-xs-offset-3 col-xs-2 text-center">
+        <span id="js-playerPoints" class="badge">
+        0
+        </span>
+        <span id="js-playerName" class="name">
+        Player Name
+        </span>
+      </div>
+      <div class="col-xs-2 text-center">
+        VS.
+      </div>
+      <div class="col-xs-2 text-center">
+        <span id="js-computerPoints" class="badge">
+        0
+        </span >
+        <span id="js-computerName" class="name">
+        Computer
+        </span>
+      </div>
+    </div>
+
+    <div class="row"> <!-- row 2 -->
+      <div class="col-xs-offset-3 col-xs-2 text-center text-center">
+        <span id="js-playerPick"></span>
+      </div>
+      <div class="col-xs-2"></div>
```

Comment:
Ok, zrobione. :)
Próbowałem wstawić pojedynczego div-a (tego z id="js-finalWinner"), ale okazało się, że w HTML5 to nie działa.